firedrake/plot.py

```diff
@@ -560,102 +413,75 @@ def _bezier_calculate_points(function):
     return np.dot(function.dat.data_ro[cell_node_list], M_inv)
 
 
-def bezier_plot(function, axes=None, **kwargs):
+def _bezier_plot(function, axes, **kwargs):
     """Plot a 1D function on a function space with order no more than 4 using
     Bezier curve within each cell, return a matplotlib axes
 
     :arg function: 1D function for plotting
-    :arg axes: Axes for plotting, if None, a new one will be created
+    :arg axes: Axes for plotting
     :arg kwargs: additional key work arguments to plot
```

Comment:
Worth updating this to note it's a firedrake function as elsewhere. Also if it's a matplotlib `Axes` object, that can be specifically noted/linked to